x-pack/auditbeat/module/system/host/host.go

```diff
@@ -29,6 +29,7 @@ import (
 const (
 	moduleName    = "system"
 	metricsetName = "host"
+	namespace     = "system.audit.host"
```

Comment:
My concern is that even with https://github.com/elastic/beats/pull/8941 merged the dataset name here will still be `system.host` but should be `system.audit.host`. Do you want to keep it `host` as the name for the config?

Reply:
Maybe we need a way to specify a custom dataset name? `{module}.{metricset}` will be weird for Auditbeat:

- auditd will be `auditd.auditd`
- file_integrity will be `file_integrity.file`

Reply:
+1. I'd say we merge this PR and make another one for the dataset?

Reply:
SGTM